spec/integration/geo_aardvark_config_spec.rb

```diff
@@ -320,7 +319,7 @@
     let(:druid) { 'bb021mm7809' }
 
     it 'maps the resource class' do
-      expect(result['gbl_resourceClass_sm']).to eq %w[Datasets Maps]
+      expect(result['gbl_resourceClass_sm']).to eq ['Datasets']
```

Comment:
Aren't we supposed to test that these are classed as both Maps and Datasets?

Reply:
Nope – this one is not georeferenced, so it's only a dataset.